articles/advisor/advisor-reference-operational-excellence-recommendations.md

```diff
@@ -524,7 +524,11 @@ Azure Front Door Standard and Premium combine the capabilities of static and dyn
 
 Learn more about [Azure Front Door (classic) will be retired on 31 March 2027](https://azure.microsoft.com/updates/azure-front-door-classic-will-be-retired-on-31-march-2027/).
 
+### Migrate Azure CDN Standard from Microsoft (Classic) to Azure Front Door Standard/Premium tier
 
+Azure CDN Standard from Microsoft (classic) will be retired on 30 September 2027. We encourage you to use the zero down-time migration tool to migrate to the Azure Front Door Standard and Premium SKUs which have feature parity with Azure CDN Standard from Microsoft (classic), as well as new features and security enhancements.
+
+Learn more about [Migrate Azure CDN from Microsoft (classic) to Standard/Premium tier](https://learn.microsoft.com/azure/cdn/migrate-tier)
```

Comment:
Replace with relative link. Check the rest of the article for examples, one I found was - "(/azure/governance/policy/overview)". 

![image](https://github.com/user-attachments/assets/1ea3db83-ab7c-4f32-85cb-f64965760ef5)

Reply:
Fixed the absolute URL